test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -1,9 +1,6 @@
+
```

Comment:
copyright should be first line

Reply:
@PaulHigin  Can you address this?

Reply:
@PaulHigin  Can you fix this?

Reply:
@TravisEz13  ?? The copyright is the first line.  AFAIK this was fixed a month ago.  In any case I don't see why the check-in should be held up for month about this...